docs/tasks/acme/issuing-certificates.rst

```diff
@@ -8,10 +8,10 @@ resource that you create in order to determine how to solve the
 
 In future releases of cert-manager, this configuration is likely to move off of
 the Certificate resource and onto the Issuer resource in order to create a
-better separation of concerns. More info can be found on issue `#XXX`_.
+better separation of concerns. More info can be found on issue `#1450`_.
 
-.. _`ACME challenges`:
-.. _`#XXX`:
+.. _`ACME challenges`: https://letsencrypt.org/how-it-works/
```

Comment:
It might be that `how-it-works` is the best reference for this use-case because its more general but I wanted to make sure you also saw we have a new docs page specifically about the challenge types: https://letsencrypt.org/docs/challenge-types/

Reply:
Thank you for this! I did look for something along these lines, but couldn't find it 😄 

I'm going to update a few other places with this new link then too.. :)

Reply:
>  I did look for something along these lines, but couldn't find it

It's brand new and might not have much "Google juice" yet.

> I'm going to update a few other places with this new link then too.. :)

 Awesome! Thanks!